modules/ROOT/pages/dataweave-selectors.adoc

```diff
@@ -128,40 +150,57 @@ payload.name
 
 ==== Output
 
+The script outputs an array that contains the values of the input objects
+with the key "name": "Arg" and "Japan".
+
 [source,json, linenums]
 ----
 ["Arg", "Japan"]
 ----
 
-
 === Example
 
-Shows how to select an xml with namespace or directly with local name
+The following example selects values from XML elements with different namespaces.
 
 ==== Input
 
+The following input payload contains XML with two `table` elements under the
+`root` element. One `table` element contains the namespace
+`"http://www.w3.org/TR/html4/"`. The other contains the namespace
+`"https://www.w3schools.com/furniture"`.
+
 [source,xml, linenums]
 ----
 <root>
 
-<h:table xmlns:h="http://www.w3.org/TR/html4/">
-  <h:tr>
-    <h:td>Apples</h:td>
-    <h:td>Bananas</h:td>
-  </h:tr>
-</h:table>
+  <h:table xmlns:h="http://www.w3.org/TR/html4/">
+    <h:tr>
+      <h:td>Apples</h:td>
+      <h:td>Bananas</h:td>
+    </h:tr>
+  </h:table>
 
-<f:table xmlns:f="https://www.w3schools.com/furniture">
-  <f:name>African Coffee Table</f:name>
-  <f:width>80</f:width>
-  <f:length>120</f:length>
-</f:table>
+  <f:table xmlns:f="https://www.w3schools.com/furniture">
```

Comment:
It is weird that the output adds the namespace attribute to all `f:`-prefixed elements but not to all the `h:` elements. Is there a reason for this behavior.